Add exemplary instruction for ASTE replay mode (#153)
precice/aste: docs/README.md

```

ASTE picks up the correct mesh files, extracts the data and passes the data to preCICE.

#### Example

> As an example, we want to replace the `Fluid` participant of our [`perpendicular-flap`](https://precice.org/tutorials-perpendicular-flap.html) tutorial by ASTE. Therefore, we first set the export tag on the fluid participant in our configuration file (see below) and run the simulation with one of the available fluid solvers

Reviewer: ```suggestion
As an example, we replace the `Fluid` participant of the [`perpendicular-flap`](https://precice.org/tutorials-perpendicular-flap.html) tutorial by ASTE. Therefore, we first set the export tag on the fluid participant in the configuration file (see below) and then run the simulation with one of the available fluid solvers (e.g. `fluid-openfoam` coupled to `solid-fenics`).
```

```xml
    <participant name="Fluid">
      ...
      <export:vtk directory="exported-meshes" />
    </participant>
```

> After the simulation finished, the directory `exported-meshes` contains now all the necessary information in order to use ASTE in our coupled simulation. We create a new directory in the `perpendicular-flap` directory called `fluid-aste` and move the `exported-meshes` into the new directory in order to run ASTE from a separate directory. Since ASTE supports only `explicit` coupling schemes, we switch from an `implicit` coupling scheme to an `explicit` coupling scheme in the preCICE configuration file as follows:

Reviewer: ```suggestion
Once the simulation is done, the directory `exported-meshes` contains all necessary coupling data in order to use ASTE for the coupled simulation. We create a new directory in the `perpendicular-flap` directory called `fluid-aste` and move the `exported-meshes` into the new directory in order to run ASTE from a separate directory. Since ASTE supports only `explicit` coupling schemes, we switch from an `implicit` coupling scheme to an `explicit` coupling scheme in the preCICE configuration file:
```

```xml
<coupling-scheme:parallel-explicit>
  <time-window-size value="0.01" />
  <max-time value="5" />
  <participants first="Fluid" second="Solid" />
  <exchange data="Force" mesh="Solid-Mesh" from="Fluid" to="Solid" />
  <exchange data="Displacement" mesh="Solid-Mesh" from="Solid" to="Fluid" />
</coupling-scheme:parallel-explicit>
```

> As a last step, we create and configure the `aste-config.json` file in the `fluid-aste` according to the data names in the `precice-config.xml` file:

Reviewer: ```suggestion
As a last step, we create and configure the `aste-config.json` file in the `fluid-aste` directory according to the data names in the `precice-config.xml` file:
```

```json
{
  "participant": "Fluid",
  "startdt": "1",
  "meshes": [
    {
      "mesh": "Fluid-Mesh",
      "meshfileprefix": "./exported-meshes/Fluid-Mesh-Fluid",
      "read-data": {
        "vector": ["Displacement"]
      },
      "write-data": {
        "vector": ["Force"]
      }
    }
  ],
  "precice-config": "../precice-config.xml"
}
```

Afterwards, the fluid solver emulation via ASTE can be started by executing

```bash
 precice-aste-run --aste-config aste-config.json
 ```

> in the `fluid-aste` directory.

Reviewer: ```suggestion
in the `fluid-aste` directory. Simply start any solid solver alongside (e.g. `solid-fenics`).
```
